shared/message/input/Buttons.proto

```diff
@@ -0,0 +1,32 @@
+/*
+* This file is part of the NUbots Codebase.
+*
+* The NUbots Codebase is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* The NUbots Codebase is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+*
+* Copyright 2023 NUbots <nubots@nubots.net>
+*/
+syntax = "proto3";
+
+package message.input;
+
+/// @author Johanne Montano
+
+message ButtonLeftDown {}
+message ButtonLeftUp {}
+
+message ButtonMiddleDown {}
+message ButtonMiddleUp {}
+
+message ButtonRightDown {}
+message ButtonRightUp{}
```

Comment:
We already have button messages, since we already have functionality for the green button, they should be in Sensors.proto I think?

Reply:
I think they're in raw sensors.
Moving them here and out of raw sensors makes sense to me. That said they should be moved, not new ones added